pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -76,9 +76,16 @@ const (
 	// MaxMonCount Maximum allowed mon count for a cluster
 	MaxMonCount = 9
 
+	// DefaultMsgr1PortName retains previous naming convention, but can be customized if needed
+    DefaultMsgr1PortName = "msgr1"
```

Comment:
Wrong indent? Do you have a linter?

Reply:
Also, it does not seem to be used elsewhere so you don't need to export it, just name it `defaultMsgr1PortName`

Reply:
removed variables and hard coded istio compliant names